Microsoft.HealthVault.Client.Android/AndroidHealthVaultConnectionFactory.cs

```diff
@@ -32,5 +35,14 @@ public static IHealthVaultConnectionFactory Current
                 }
             }
         }
+
+        public static IThingTypeRegistrar ThingTypeRegistrar
```

Comment:
I will add a TODO on this class

---
In reply to: [123317884](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/62#discussion_r123317884) [](ancestors = 123317884)